cmd/manager/main.go

```diff
@@ -203,7 +203,8 @@ func newConfigManagedClient(mgr manager.Manager) (runtimeclient.Client, manager.
 		Scheme: mgr.GetScheme(),
 		Mapper: mgr.GetRESTMapper(),
 	}
-	client, err := manager.NewClientBuilder().Build(cache, mgr.GetConfig(), clientOpts)
+
+	client, err := runtimeclient.New(config.GetConfigOrDie(), clientOpts)
```

Comment:
This client will be uncached, where the previous client is cached, I'm not sure this is correct

Reply:
Perhaps you need to use a `client.NewDelegatingClient` instead?

Reply:
like this?
```
	client, err := runtimeclient.New(config.GetConfigOrDie(), clientOpts)
	if err != nil {
		return nil, nil, err
	}

	delegatedClient, err := runtimeclient.NewDelegatingClient(runtimeclient.NewDelegatingClientInput{
		Client:      client,
		CacheReader: cache,
	})
```

Reply:
Yep, that's the one, that's basically what https://github.com/kubernetes-sigs/controller-runtime/blob/7ef2da0bc161d823f084ad21ff5f9c9bd6b0cc39/pkg/cluster/cluster.go#L259 does, perhaps could use it instead?